src/utils/grammar.js

```diff
@@ -0,0 +1,20 @@
+import * as _ from 'lodash';
+
+export const joinGrammaticallyListOfItems = items => {
+  const result = items.join(', ');
+  const lastCommaIdx = result.lastIndexOf(',');
+
+  return items.length > 1 && lastCommaIdx >= 0
+    ? `${result.substr(0, lastCommaIdx)} and${result.substr(lastCommaIdx + 1)}`
+    : result;
+};
+
+export const makeSentence = (sentence, capitalize = true) => {
+  const result = capitalize ? _.capitalize(sentence) : sentence;
+  return !result || result.charAt(result.length) === '.' ? result : `${result}.`;
+};
+
+export const addMissingSubject = (sentence, subject) => {
+  const c = sentence ? sentence.charAt(0) : '';
+  return c.toLowerCase() !== c ? sentence : `${_.capitalize(subject)} ${sentence}`;
```

Comment:
Vivat future i18n effort :-(

Reply:
This feature is already implemented in both fork and `openshift/console`. It just doesn't have its own function. 

i18n will be resolved once its time comes